From 5254285636a2d58805152be0f9700f773624879c Mon Sep 17 00:00:00 2001
From: ami_stuff <ami_stuff@o2.pl>
Date: Wed, 11 May 2011 22:18:15 +0200
Subject: [PATCH 04/57] riff: Add DAVC fourcc.

This fourcc is used by the "mpegable AVC" codec and files encoded with
this codec decode correctly with our H.264 decoder.
(cherry picked from commit 2ea1ca1714f655a463b941b115e45a1a2f031f7b)

Signed-off-by: Reinhard Tartler <siretart@tauware.de>
---
 libavformat/riff.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libavformat/riff.c b/libavformat/riff.c
index 95e08b2..b2e8414 100644
--- a/libavformat/riff.c
+++ b/libavformat/riff.c
@@ -33,6 +33,7 @@ const AVCodecTag ff_codec_bmp_tags[] = {
     { CODEC_ID_H264,         MKTAG('X', '2', '6', '4') },
     { CODEC_ID_H264,         MKTAG('x', '2', '6', '4') },
     { CODEC_ID_H264,         MKTAG('a', 'v', 'c', '1') },
+    { CODEC_ID_H264,         MKTAG('D', 'A', 'V', 'C') },
     { CODEC_ID_H264,         MKTAG('V', 'S', 'S', 'H') },
     { CODEC_ID_H263,         MKTAG('H', '2', '6', '3') },
     { CODEC_ID_H263,         MKTAG('X', '2', '6', '3') },
-- 
1.7.4.1

